open-sphere-base/core/src/main/java/io/opensphere/core/quantify/Quantify.java

```diff
@@ -0,0 +1,84 @@
+package io.opensphere.core.quantify;
+
+import io.opensphere.core.Toolbox;
+
+public class Quantify
+{
+    /** A singleton initalized toolbox used to access application state. */
+    private static volatile Toolbox ourToolbox;
```

Comment:
Can you store the plugin toolbox or the quantify service here instead of the toolbox, or are they not available yet when setToolbox is called?

Reply:
They're not yet available when the set toolbox is called. I followed your pattern and initialized the toolbox from the kernel.